application/clicommands/ProcessCommand.php

```diff
@@ -72,6 +74,10 @@ public function listAction()
      *   --config <configname>   Name of the config that contains <process>
      *   --details               Show problem details as a tree
      *   --colors                Show colored output
+     *   --htmlcolors            Show colored output as htmltable with links
+     *                           to the displayed hosts. You need to set your
```

Comment:
"to the displayed *hosts*"? Aren't services linked too? What about "to the displayed *nodes*"?